src/YunInternet/Libvirt/Configuration/Domain/Device/Disk/backingStore.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * Date: 19-2-17
+ * Time: 上午1:20
+ */
+
+namespace YunInternet\Libvirt\Configuration\Domain\Device\Disk;
+
+use YunInternet\Libvirt\XMLImplement\SimpleXMLImplement;
+use YunInternet\Libvirt\XMLImplement\SingletonChild;
+
+/**
+ * Class backingStore
+ * @method XMLElementContract source()
+ * @method XMLElementContract format()
+ * @package YunInternet\Libvirt\Configuration\Domain\Device\Disk
+ */
+class backingStore extends SimpleXMLImplement
+{
+    protected $singletonChildWrappers = [
+        "backingStore" => \YunInternet\Libvirt\Configuration\Domain\Device\Disk\backingStore::class
+    ];
+
+    use SingletonChild;
+
+    public function isActive()
+    {
+        return !empty($this->getType());
+    }
+
+    public function Nested(int $num = 0)
+    {
+        if ($this->backingStore()->isActive())
+        {
+            $num = $this->backingStore()->Nested($num);
```

Comment:
You should use $num += $this->backingStore()..., rather than require a parameter.